egs/libri_css/asr1_mono/diarization/VB_resegmentation.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
```

Comment:
where is this script used in this recipe?

Reply:
I've removed this and few more unused scripts.